### PR TITLE
feat: useScrollbarWidth hook;

### PR DESCRIPTION
**Pull request by xobotyi**
# Description

This hook uses [@xobotyi/scrollbar-width](https://github.com/xobotyi/scrollbar-width) underneath, which is well tested and grown from my scrollbars component.

This PR absorbs #653 

## Type of change

- [ ] Bug fix _(non-breaking change which fixes an issue)_
- [x] New feature _(non-breaking change which adds functionality)_
- [ ] **Breaking change** _(fix or feature that would cause existing functionality to not work as before)_

# Checklist
- [x] Read the [Contributing Guide](https://github.com/streamich/react-use/blob/master/CONTRIBUTING.md)
- [x] Perform a code self-review
- [x] Comment the code, particularly in hard-to-understand areas
- [x] Add documentation
- [x] Add hook's story at Storybook
- [x] Cover changes with tests
- [x] Ensure the test suite passes (`yarn test`)
- [x] Provide 100% tests coverage
- [x] Make sure code lints (`yarn lint`). Fix it with `yarn lint:fix` in case of failure.
- [x] Make sure types are fine (`yarn lint:types`).

### CODE REVIEW

**Comment by xobotyi** [comment]
@streamich the only thing im not sure in - should it be exported by default and be a default dependency or make it optional?
@xobotyi/scrollbar-width is [tiny enough](https://bundlephobia.com/result?p=@xobotyi/scrollbar-width@1.4.1) so can be exported as default, but it is on you to decide.

**Comment by streamich** [comment]
@xobotyi As I understand it can return `undefined` only when called before DOM is ready. What would be a case when React renders and DOM is not ready yet?

**Comment by xobotyi** [comment]
@streamich if developer added react bundle before the closing `body` tag and/or placed in iframe.
Had several issues related to that behavior: [#115](https://github.com/xobotyi/react-scrollbars-custom/issues/115) and [#123](https://github.com/xobotyi/react-scrollbars-custom/issues/123)

File: package.json [hunk 1]
```diff
@@ -78,6 +79,7 @@
     "@testing-library/react-hooks": "3.2.1",
     "@types/jest": "24.0.23",
     "@types/react": "16.9.11",
+    "@xobotyi/scrollbar-width": "1.5.0",
```
Comment (streamich) [inline package.json]: We already have it in `"dependencies"`, probably this line is not needed.

Comment (xobotyi) [reply package.json]: Forgot to remove from dev-deps.

**Pull request merged**